content/en/docs/concepts/architecture/leases.md

```diff
@@ -1,5 +1,9 @@
 ---
 title: Leases
+api_metadata:
+  apiVersion: "coordination.k8s.io/v1"
+  kind: "Lease"
+  type: cluster
```

Comment:
I'd make this a _list_ (because a concept might apply to something with two stable API versions, such as  HorizontalPodAutoscaler or Event)

eg (note the extra initial `-`)
```diff
-  apiVersion: "coordination.k8s.io/v1"
+- apiVersion: "coordination.k8s.io/v1"
   kind: "Lease"
-  type: cluster
+  api_reference_section: cluster
```

Reply:
If you want a hint about how to work out the `api_reference_section` automatically, let me know. I can help.

Reply:
I've made changes to implement this in the recent commits.